js/src/forum/components/PostStreamScrubber.js

```diff
@@ -65,6 +77,18 @@ export default class PostStreamScrubber extends Component {
             </a>
 
             <div className="Scrubber-scrollbar">
+              {showBackButton ? (
+                <a
+                  style={'top: ' + this.percentPerPost().index * this.stream.previousIndex + '%'}
+                  className="Scrubber-back"
+                  onclick={this.returnToLastPosition.bind(this)}
+                  title={app.translator.trans('core.forum.post_scrubber.back_title')}
```

Comment:
The translation for this should be added to this repo, as there are no more translations in the lang english repo. Also, a jquery tooltip might look better, although I'm not 100% sure that it's worth the pain of getting it to work.